Makefile

```diff
@@ -1,70 +1,63 @@
 REGISTRY       ?= quay.io
-ORG            ?= zvonkok
-TAG            ?= latest #$(shell git rev-parse --short HEAD)
+ORG            ?= openshift-psap
+TAG            ?= $(shell git branch | grep \* | cut -d ' ' -f2)
```

Comment:
Are you trying to get current branch here?  Won't `git rev-parse --abbrev-ref HEAD` work?

Reply:
You're right, done.